### PR TITLE
Added up the Contibuting.md and code of conduct for contribution guidelines.

### PR DESCRIPTION
**Pull request by abhay-raj19**
Fixes up the issue #32

### CODE REVIEW

**Comment by abhay-raj19** [comment]
@TanmayDhobale @hkirat is it fine sir ?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```
Comment (TanmayDhobale) [inline CODE_OF_CONDUCT.md:1]: no need

Comment (abhay-raj19) [reply CODE_OF_CONDUCT.md:1]: It is need though if we open-source a codebase, but I can remove it on your choice ?
What do u say sir ?

Comment (TanmayDhobale) [reply CODE_OF_CONDUCT.md:1]: remove that file only

**Comment by abhay-raj19** [comment]
Removed up as said @TanmayDhobale

**Comment by devsargam** [comment]
Thanks!!!

**Comment by tanmay4l** [comment]
> Thanks!!!

/bounty ?

**Comment by abhay-raj19** [comment]
> > Thanks!!!
> 
> /bounty ?

I was also thinking up same @TanmayDhobale @devsargam

**Comment by tanmay4l** [comment]
> > > Thanks!!!
> > 
> > /bounty ?
> 
> I was also thinking up same @TanmayDhobale @devsargam 

Bro chill I am kidding

**Comment by tanmay4l** [comment]
> > > > Thanks!!!
> > > 
> > > /bounty ?
> > 
> > I was also thinking up same @TanmayDhobale @devsargam 
> 
> Bro chill I am kidding 

@abhay-raj19 contribute to other repo u will get there

**Comment by Akshat76845** [comment]
kl

**Pull request merged**